runtime-doc/poppler/autobuild/prepare

```diff
@@ -1,4 +1 @@
-ln -sv /usr/lib/qt4/bin/moc "$SRCDIR"/moc-qt4
-export PATH="${SRCDIR}:$PATH"
-
 export CXXFLAGS="${CXXFLAGS} -I/usr/include/openjpeg-2.3"
```

Comment:
abinfo here.

Reply:
The extra CXXFLAGS is no longer required: upstream can automatically detect openjpeg.